kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -167,6 +167,12 @@ data:
 
 # Doc - https://github.com/microsoft/Docker-Provider/blob/ci_prod/Documentation/AgentSettings/ReadMe.md
   agent-settings: |-
+    # High log scale collection mode is used for high log volume scenarios. It is recommended to enable this mode only if you are experiencing high log volume issues since this in Public preview.
+    # Refer to public documentation for more details - https://aka.ms/HighLogScaleMode before enabling this setting
+    # Please note, this setting has additional dependencies such as Data Collection endpoint and Microsoft-ContainerLogV2-HighScale stream instead of Microsoft-ContainerLogV2.
+    [agent_settings.high_log_scale]
+      enabled = true
```

Comment:
is this enabled by default if configmap is applied? What if customer want to tweak other features